flow_test.go

```diff
@@ -169,6 +171,49 @@ func TestUnregister(t *testing.T) {
 	wg.Wait()
 }
 
+func TestMockClock(t *testing.T) {
+	mockClock := useMockClock()
+	defer RestoreClock()
+
+	m := new(Meter)
+	for i := 0; i < 300; i++ {
+		m.Mark(1000)
+		mockClock.Add(40 * time.Millisecond)
+	}
+	actual := m.Snapshot()
+	checkApproxEq(t, actual.Rate, 25000, 100)
```

Comment:
I think I've said this before, but there's no need for approximations. We're using a mock clock, so timing is precise, and so should be the measurement.